testing/scripts/test_prepackaged_servers.py

```diff
@@ -90,3 +91,27 @@ def test_mlflow(self, namespace):
         assert r.status_code == 200
 
         run(f"kubectl delete -f {spec} -n {namespace}", shell=True)
+
+    # Test prepackaged Tabular SKLearn Alibi Explainer
```

Comment:
Small typo in the comment: not `Tabular`, but `Text` explainer.

Reply:
Nice one